tools/benchmark/proto/cosmos/benchmark/module/v1/module.proto

```diff
@@ -16,16 +16,16 @@ message Module {
 // GenesisParams defines the genesis parameters for the benchmark module.
 message GeneratorParams {
   // seed is the seed for the random number generator.
-  uint64 seed         = 1;
+  uint64 seed = 1;
```

Comment:
doesn't need to be committed

Reply:
It does. `make proto-gen` runs linting of files. This means when this file was committing linting wasn't run on them. The entirety of the result of `make proto-gen` should always be committed.